Formula/y/yazi.rb

```diff
@@ -0,0 +1,28 @@
+class Yazi < Formula
+  desc "️Blazing fast terminal file manager written in Rust, based on async I/O"
+  homepage "https://github.com/sxyazi/yazi"
+  url "https://github.com/sxyazi/yazi/archive/refs/tags/v0.1.3.tar.gz"
+  sha256 "e6738a12896ff0ab081a306e6334702a93dcb1039e4c596eb6ee723a6bc037cc"
+  license "MIT"
+  head "https://github.com/sxyazi/yazi.git",
+       branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "fd"
+  depends_on "ffmpegthumbnailer"
+  depends_on "fzf"
+  depends_on "jq"
+  depends_on "poppler"
+  depends_on "ripgrep"
+  depends_on "unar"
+  depends_on "zoxide"
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "app")
+    bin.install "./target/release/yazi"
```

Comment:
we probably dont need this.

Reply:
Because [`std_cargo_args`](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1634-1636) already makes it install to the right place.